docker-compose.yml

```
@@ -11,7 +11,7 @@ services:
       - DB_HOST=db
 
   db:
-    image: mongo:4.2
+    image: mongo:5.0
```

```suggestion
    image: mongo:4.4
```
Maybe we could keep the minimum supported version here, to avoid incompatibility in new features.